samples/subsys/usb/cdc_acm/src/main.c

```diff
@@ -52,10 +52,24 @@ static inline void print_baudrate(const struct device *dev)
 static struct usbd_contex *sample_usbd;
 K_SEM_DEFINE(dtr_sem, 0, 1);
 
-static void sample_msg_cb(const struct usbd_msg *msg)
+static void sample_msg_cb(struct usbd_contex *const ctx, const struct usbd_msg *msg)
 {
 	LOG_INF("USBD message: %s", usbd_msg_type_string(msg->type));
 
+	if (usbd_can_detect_vbus(ctx)) {
+		if (msg->type == USBD_MSG_VBUS_READY) {
+			if (usbd_enable(ctx)) {
+				LOG_ERR("Failed to enable device support");
+			}
+		}
+
+		if (msg->type == USBD_MSG_VBUS_REMOVED) {
```

Comment:
```suggestion
		} else if (msg->type == USBD_MSG_VBUS_REMOVED) {
```

Should there be an additional check/log statement if the type is neither of those types?

Reply:
> Should there be an additional check/log statement if the type is neither of those types?

No, in this block we do not care about others.